specs/interop/upgrade.md

```diff
@@ -26,6 +29,43 @@ The execution payload MUST set `noTxPool` to `true` for this block.
 
 The exact definitions for these upgrade transactions are still to be defined.
 
+## L1 Attributes
```

Comment:
Here did you confuse `setL1BlockValuesInterop` with `setL1BlockIsthmus`?

Reply:
yes, this is outdated